lighthouse-core/scripts/release/prepare-commit.sh

```diff
@@ -40,11 +40,8 @@ git checkout -b "$BRANCH_NAME"
 # Install the dependencies.
 yarn install
 
-# Bump the version in package.json
-NEEDLE="^  \"version\": \"$SEMVER_PATTERN\""
-REPLACEMENT="  \"version\": \"$NEW_VERSION\""
-
-sed -i '' "s/$NEEDLE/$REPLACEMENT/g" package.json
+# Bump the version in package.json and others.
```

Comment:
nice 👍

Reply:
and the release script rewrite has begun >:)